Documentation/coding-guidelines/breaking-changes.md

```diff
@@ -23,14 +23,23 @@ To help triage breaking changes, we classify them in to four buckets:
 4. Clearly Non-Public
 
 ## Bucket 1: Public Contract
-*Clear violation of public contract.*
+*Clear [violation][breaking-change] of public contract.*
```

Comment:
💭 I'm torn on this link being applied to just 'violation'. It seems almost like it should apply either to 'public contract' or to the complete text 'violation of public contract'.